src/terraform/index.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // terraformModulesCommitHash is the git commit hash that reflects
 // which of the Terraform modules in Originate/exosphere we are using
-const terraformModulesCommitHash = "8786f912"
+const terraformModulesCommitHash = "48731e6"
```

Comment:
We can write two small scripts. One that verifies this value if the latest commit that changed anything in terraform modules. Another that updates it to the latest. We would probably use it in the release process.

Alternatively do we need to have a commit hash? Is there another way we could get by without it?